net-fs/davfs2/davfs2-1.5.2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools linux-mod user
+
+DESCRIPTION="Linux FUSE (or coda) driver that allows you to mount a WebDAV resource"
+HOMEPAGE="http://savannah.nongnu.org/projects/davfs2"
+SRC_URI="http://mirror.lihnidos.org/GNU/savannah/davfs2/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+RESTRICT="test"
+
+DEPEND="dev-libs/libxml2
+	net-libs/neon
+	sys-libs/zlib"
+RDEPEND="${DEPEND}"
+
+pkg_setup() {
+	enewgroup davfs2
+}
+
+src_prepare() {
+	eautoreconf
+}
+
+src_configure() {
+	econf dav_user=nobody --enable-largefile --docdir=/usr/share/doc/${P}
```

Comment:
If i may nitpick a bit, it's usually a good idea to use \ here and pass one parameter per line.